homeassistant/components/bosch_shc/sensor.py

```diff
@@ -206,7 +204,6 @@ class PuritySensor(SHCEntity, SensorEntity):
     def __init__(self, device: SHCDevice, parent_id: str, entry_id: str) -> None:
         """Initialize an SHC purity reporting sensor."""
         super().__init__(device, parent_id, entry_id)
-        self._attr_name = f"{device.name} Purity"
```

Comment:
What provides the "Purity" name here now?

Reply:
It seems for me not all sensors have been applied with a translation key. @joostlek will you add those missing, otherwise please do not remove the `self._attr_name` attribute.

Reply:
Yes, will do, I already shut down my pc, so tomorrow I will fix this